adapters/ing-adapter/src/main/java/de/adorsys/xs2a/adapter/ing/IngMapper.java

```diff
@@ -0,0 +1,342 @@
+package de.adorsys.xs2a.adapter.ing;
+
+import de.adorsys.xs2a.adapter.api.model.*;
+import de.adorsys.xs2a.adapter.ing.model.*;
+import de.adorsys.xs2a.adapter.service.model.TokenResponse;
+import org.mapstruct.InheritInverseConfiguration;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+@Mapper
+public interface IngMapper {
+    AccountList map(IngAccountsResponse value);
```

Comment:
Can't find teste for this mapper, does it exist?

Reply:
no

Reply:
please add it

Reply:
added